### PR TITLE
Load the parents eagerly

### PR DESCRIPTION
**Pull request by pmverma**
Partial revert the #10667 
Fixes the problem described https://github.com/jhipster/generator-jhipster/pull/10667#issuecomment-552960466

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by cbornet** [comment]
It should also be done for OneToOne

File: generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs [hunk 1]
```diff
@@ -352,14 +352,14 @@ public class <%= asEntity(entityClass) %> implements Serializable {
...
-    @OneToOne(fetch = FetchType.LAZY<% if (relationshipRequired) { %>, optional = false<% } %>)
+    @OneToOne<% if (relationshipRequired) { %>(optional = false)<% } %>
             <%_ if (relationshipValidate) { _%>
     <%- include relationship_validators -%>
             <%_ }_%>
             <%_ if (isUsingMapsId === true) { %>
     @MapsId
     @JoinColumn(name = "id")
-            <%_ } else { %>
+            <%_ } else { _%>
```
Comment (pascalgrimaud) [inline generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs]: nice eagle eyes

**Comment by amirensit** [comment]
Any idea what is the purposes behind removing lazy loading from manyToOne and oneToOne relations ?

**Comment by cbornet** [comment]
That's because our front expects that the XXXToOne is eagerly loaded. If we don't do it at the hibernate level, it must be done somewhere else. Probably at the repository level.

**Pull request merged**